main.go

```diff
@@ -668,8 +682,27 @@ func clientTransport(ctx *cli.Context, enableTLS bool) http.RoundTripper {
 			ClientSessionCache:       tls.NewLRUClientSessionCache(tlsClientSessionCacheSize),
 		}
 	}
+	rtw.transport = tr
+}
 
-	return tr
+func (rtw *roundTripperWrapper) RoundTrip(req *http.Request) (*http.Response, error) {
+	rtw.rwLocker.RLock()
+	isNil := rtw.transport == nil
+	rtw.rwLocker.RUnlock()
+	if isNil {
+		rtw.renew()
+	}
```

Comment:
Isn't it more appropriate to use [`sync.Once`](https://pkg.go.dev/sync#Once) instead?